flake.nix

```diff
@@ -12,19 +14,8 @@
       flake = false;
     };
 
-    ogmios.url = "github:mlabs-haskell/ogmios/3b229c1795efa30243485730b78ea053992fdc7a";
-
-    plutip.url = "github:mlabs-haskell/plutip/1c9dd05697d7cf55de8ca26f0756a75ed821bdfb";
-    plutip.inputs.bot-plutus-interface.follows = "bot-plutus-interface";
```

Comment:
I am not good at Nix, can you explain why all these got delted?

Reply:
Upstream got updated, they are no longer needed